app/controllers/organizations_controller.rb

```diff
@@ -16,19 +16,13 @@ def edit
 
   def show
   end
-  
-  # # POST /organizations
-  # def create
-  #   @organization = Organization.new(organization_params)
-  #   @organization.users << current_user
-  #   flash[:notice] = 'Organization was successfully created.' if @organization.save
-  #   respond_with(@organization) # In order to use respond_with, first you need to declare the formats your controller responds to in the class level.
-  # end
 
   def create
     @organization = Organization.new(organization_params)
     @organization.users << current_user
-
+    if (organization_params[:attachment])
```

Comment:
I think that it would be better to move this verification inside `add_attachment` method.